OloEngine/src/OloEngine/Renderer/MeshOptimization.cpp

```
@@ -0,0 +1,711 @@
+
+        // Vertex fetch analysis
+        meshopt_VertexFetchStatistics const fetchStats =
+            meshopt_analyzeVertexFetch(indices.GetData(), indexCount, vertexCount, sizeof(Vertex));
+        result.OverfetchRatio = fetchStats.overfetch;
+
+        return result;
+    }
+
+    // ── Meshlet generation ─────────────────────────────────────────
+
+    MeshletData GenerateMeshlets(const MeshSource& meshSource, u32 maxVertices, u32 maxTriangles)
```

Cppcheck (2.20.0)

[style] The function 'GenerateMeshlets' is never used.

(unusedFunction)